From 46c39a621a5f2ed24f27b8db9b90fc92a9d0ae89 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Wed, 6 Oct 2010 21:01:17 +0900
Subject: [PATCH 1/2] Use full path for etckeeper in commit-hook

This allows us to use etckeeper in cron script etc. where PATH
does not include /usr/sbin.
---
 init.d/50vcs-pre-commit-hook |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/init.d/50vcs-pre-commit-hook b/init.d/50vcs-pre-commit-hook
index e1745b1..c7a354f 100755
--- a/init.d/50vcs-pre-commit-hook
+++ b/init.d/50vcs-pre-commit-hook
@@ -27,7 +27,7 @@ EOF
 			cat >>.hg/hgrc <<EOF
 [hooks]
 # pre-commit hook for etckeeper, to store metadata and do sanity checks
-precommit = etckeeper pre-commit -d `pwd`
+precommit = /usr/sbin/etckeeper pre-commit -d `pwd`
 EOF
 		fi
 	;;
-- 
1.7.1

